settings/security/ssl-certificates.rst

```diff
@@ -0,0 +1,86 @@
+SSL Certificates
+****************
+
+Introduction
+------------
+
+Zammad offers the possibility to upload custom certificates and custom
+Certificate Authority (CA) certificates. This can be useful if you want to
+connect Zammad the secure way to other systems which are using custom
+certificates.
+
+An example usecase might be: You have an internal LDAP server that is not
+accessible from the internet, and you want a SSL-encrypted connection, using a
+custom certificate.
+
+Prerequisites
+-------------
+
+You must have an existing certificate and/or a custom CA certificate, you want
+to add to Zammad. As filetype ``.crt`` is supported and the certificate format
+has to be PEM (Base64 ASCII).
+
+.. hint::
+
+    When you import a custom CA certificate, all certificates generated with
+    this are trusted. This can be useful if you have more than one
+    system/certificate you want to connect to and the certificates are issued
+    from the same custom CA.
+
+Add a Certificate
+-----------------
+
+To add a custom certificate or a custom CA certificate, head over to the Admin
+panel > Settings > Security > SSL Certificates. Here you can find a "Add SSL
+Certificate" button. In the dialog you can either select a certificate file
+or paste the content of the certificate:
+
+.. figure:: /images/settings/security/ssl-add-certificate-dialog.png
+    :align: center
+    :scale: 60 %
+    :alt: Screenshot showing certificate adding dialog
+
+After selecting a file or pasting the content of the certificate, click on the
```

Comment:
Maybe it would be helpful to state that after uploading, the certificates are immediately effective, no restarts needed etc.